recipes/qutip/bld.bat

```diff
@@ -0,0 +1,10 @@
+conda clean --yes --source-cache
```

Comment:
You don't need this.  In my package I had that because the upstream name comes from mercurial and all sources are named `tip.tar.gz`, so different software have the same name and cleaning the cache is important.

Here, not only `qutip` is a unique name, but when it becomes a feedstock this won't matter.